src/index.js

```diff
@@ -1,16 +1,35 @@
 import './style.css';
+import * as allVar from './modules/variables.js';
 
-const data = [
-  { name: 'Ahmed', score: 90 },
-  { name: 'Ali', score: 80 },
-  { name: 'David', score: 99 },
-  { name: 'Alexender', score: 95 },
-];
-
-const display = () => {
+const display = (data) => {
   data.forEach((i) => {
-    document.querySelector('.infos').innerHTML += `<p class="info">${i.name} : ${i.score} </p>`;
+    allVar.List.innerHTML += `<p class="info">${i.user} : ${i.score} </p>`;
   });
 };
 
-display();
\ No newline at end of file
+fetch(allVar.BASE_URL)
+  .then((res) => res.json())
+  .then((data) => display(data?.result));
+
+allVar.Form.addEventListener('submit', async (e) => {
+  e.preventDefault();
+  const userObj = {
+    user: e.target.user.value,
+    score: e.target.score.value,
+  };
+
+  const postRequest = {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify(userObj),
+  };
+
+  await fetch(allVar.BASE_URL, postRequest).then((res) => res.json())
+    .then(() => window.location.reload());
+});
+
+allVar.Refresh.addEventListener('click', () => {
+  window.location.reload();
```

Comment:
- [ ] When the refresh button is clicked, the requirement is that it should hit the GET API endpoint causing the scores list to change. But you are simply reloading the page when the button is clicked. Please try to implement this requirement 👍 
![image](https://user-images.githubusercontent.com/46839548/225756132-197684a7-a50b-4c49-bf74-f5aba02924e0.png)